config/samples/backends/ceph_thin_provisioning/ceph.yaml

```diff
@@ -0,0 +1,50 @@
+# Sample using Ceph as a glance backend
```

Comment:
Given that this sample is a variation of the basic `backend/ceph` (we are only adding the `rbd_thin_provisioning = True` config option), I think we can remove this file and point to the other one via `kustomization.yaml`. This would reduce duplication and we only have to maintain a basic Ceph sample.